src/Microsoft.TemplateEngine.Core/Conditional.cs

```diff
@@ -308,6 +472,25 @@ public bool BranchTaken
                     get { return _branchTaken; }
                     set { _branchTaken |= value; }
                 }
+
+                public bool ActionableOperationsEnabled { get; private set; }
+
+                public void ToggleActionableOperations(bool enabled, IProcessorState processor)
+                {
+                    ActionableOperationsEnabled = enabled;
+
+                    foreach (string otherOptionDisableFlag in _impl._definition._actionableOperations)
```

Comment:
Right… no clue what I was thinking.

From: Mike Lorbetske [mailto:notifications@github.com]
Sent: Tuesday, August 23, 2016 11:14 AM
To: seancpeters/microsoft-templateengine <microsoft-templateengine@noreply.github.com>
Cc: Sean Peters <sepeters@microsoft.com>; Author <author@noreply.github.com>
Subject: Re: [seancpeters/microsoft-templateengine] Refactor (#1)


In src/Microsoft.TemplateEngine.Core/Conditional.cs<https://github.com/seancpeters/microsoft-templateengine/pull/1#discussion_r75921414>:

> @@ -308,6 +472,25 @@ public bool BranchTaken

>                      get { return _branchTaken; }

>                      set { _branchTaken |= value; }

>                  }

> +

> +                public bool ActionableOperationsEnabled { get; private set; }

> +

> +                public void ToggleActionableOperations(bool enabled, IProcessorState processor)

> +                {

> +                    ActionableOperationsEnabled = enabled;

> +

> +                    foreach (string otherOptionDisableFlag in _impl._definition._actionableOperations)

processor.Config.Flags[otherOptionDisableFlag] = enabled;

instead of contains/add vs set

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/seancpeters/microsoft-templateengine/pull/1/files/a425b2f9e55c8aa446463bd90060cb6bc99db3d0#r75921414>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AT8a8V1PN7stGhX4QOvowfE_X_2Ga7IHks5qizhagaJpZM4JqaeQ>.